autogpt_platform/backend/backend/server/v2/store/media.py

```diff
@@ -67,7 +68,7 @@ async def upload_media(
     # Validate file signature/magic bytes
     if file.content_type in ALLOWED_IMAGE_TYPES:
         # Check image file signatures
-        if content.startswith(b"\xFF\xD8\xFF"):  # JPEG
+        if content.startswith(b"\xff\xd8\xff"):  # JPEG
```

Comment:
This isn’t a perfect restriction If I recall correctly. I think this can be a shared file type due to the possible contents with like mp3